tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -699,22 +697,3 @@ def test_simple(self):
             event=audit_log.get_event_id("ALERT_RULE_REMOVE"), target_object=self.alert_rule.id
         )
         assert len(audit_log_entry) == 1
-
-    def test_snapshot_and_create_new_with_same_name(self):
```

Comment:
I removed this because it's duplicated in the organization alert rule details delete tests.